neurom/features/neuritefunc.py

```diff
@@ -530,10 +534,21 @@ def neurite_volume_density(neurites, neurite_type=NeuriteType.all):
 
     The volume density is defined as the ratio of the neurite volume and
     the volume of the neurite's enclosing convex hull
+
+    .. note:: Returns `np.nan` if the convex hull computation fails.
     """
     def vol_density(neurite):
         """Volume density of a single neurite."""
-        return neurite.volume / convex_hull(neurite).volume
+        try:
+            volume = convex_hull(neurite).volume
+        except scipy.spatial.qhull.QhullError as e:
+            L.exception('Failure to compute neurite volume using the convex hull. '
+                        'Feature `neurite_volume_density` will return `np.nan`.\n'
+                        'Caught stacktrace:\n %s',
+                        e)
```

Comment:
Minor, if we log with `L.exception` I think we could avoid logging `Caught stacktrace` and the exception itself.
Anyway I didn't see a real example in this case, so if you are happy with the message I'm fine.